lib/qs.rb

```diff
@@ -129,6 +136,12 @@ class Config
       option :timeout,  Integer, :default => 1
       option :size,     Integer, :default => 4
     end
+
+    attr_accessor :dispatcher_queue_class
```

Comment:
@kellyredding - I made this a accessor on the config class instead of an option because of issues I ran into related to this issue: https://github.com/redding/ns-options/issues/84 and ns-options cloning an option that is a `Class`.